frontend/src/metabase/visualizations/echarts/tooltip/index.tsx

```diff
@@ -72,8 +72,8 @@ export const getTooltipBaseOption = (
         container = document.createElement("div");
         container.classList.add("echarts-tooltip-container");
         container.style.setProperty("overflow", "hidden");
-        container.style.setProperty("height", "100%");
-        container.style.setProperty("position", "relative");
+        container.style.setProperty("position", "absolute");
```

Comment:
The problem is that when we use iframe resizer we set `EmbedFrame` to `position: absolute; inset: 0;`

https://github.com/metabase/metabase/blob/27a91e64fcbf4e6e43f23c18b3d207bf258b1334/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx#L23-L31

This makes the `#root` div have a height greater than 0. When we don't use the iframe resizer this `#root` div has no height, so this echart tooltip container sit at the top of the iframe.

See this [message on Slack](https://metaboat.slack.com/archives/C063Q3F1HPF/p1731492251517089?thread_ts=1731311131.911449&cid=C063Q3F1HPF) for more details.

I fix the issue by ensuring the echart tooltip covers the entire iframe. 

![image](https://github.com/user-attachments/assets/ce8d0b1a-38d3-4d0c-8fe9-48d4dd80bcbc)

Compare to the version before this fix
![image](https://github.com/user-attachments/assets/795cc95a-705e-4101-8edc-a8a90f8b9744)